### ISSUE TITLE
lasot dataset

### ISSUE THREAD

**Issue by woaixueyingying** [hekaijie123/TATrack#8]
Hello, is the lasot data set to be divided into training and testing sets？

**Comment by hekaijie123** [comment]
The dataset is officially divided into a test set and a training set. The training set is used automatically when the code is trained and the test set is used when it is tested.
